backend/routes/createEmailDomainRouter.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const { createEmailDomainFactory } = require('../controllers/addEmailDomain')
+
+const router = express.Router({ mergeParams: true })
+const controller = ({ manageUsersApi }) => {
+  const createEmailDomainApi = manageUsersApi.createEmailDomain
+
+  const { index: createEmailIndex, post: createEmailExec } = createEmailDomainFactory(
+    createEmailDomainApi,
+    '/create-email-domain',
+    '/email-domains',
+  )
+
+  router.get('/', createEmailIndex)
+  router.post('/', createEmailExec)
+
```

Comment:
naming is a bit confusing, possibly createEmailDomainXX

Reply:
Done